### DISCUSSION TITLE
Scheduling repeated tasks

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#15799] [Questions]
Bear with me here, i am new to PostGres and supabase (MsSQL background), figured out most stuff, but:

Q. How do i schedule tasks/functions to run at intervals (every hour) over the DB (would be sql agent jobs in MsSql)... i cant see anything like that?

**Comment by GaryAustin1** [comment]
The pg_cron extension is what you want.
https://supabase.com/docs/guides/database/extensions/pg_cron#syntax
